spec/requests/app_spec.rb

```diff
@@ -12,4 +12,28 @@
       'P4L-email' => user.email
     }
   end
+
+  describe 'GET /home' do
+    include ActiveSupport::Testing::TimeHelpers
+    it 'returns the correct verse' do
+
+      # year, month, day, hour, minute, second
+      travel_to Time.local(2024, 4, 9, 1, 0, 0) do
+        verse = create(:verse, day: 1)
+        get '/home', {}, headers
+
+        expect(last_response.status).to eq(200)
+        expect(parsed_response).to match(data_object_for(verse))
+
+      end
+
+      travel_to Time.local(2024, 4, 10, 1, 0, 0) do
```

Comment:
⛏️ Tiny nitpick here, you've already tested the logic of the verse of the day so you don't need to repeat it here, we're more concerned with the API response in this context, which is covered by the 200 status and the data returned by the first one.

Reply:
Gotcha! 🫡